### PR TITLE
feat: remove absolute log paths from log_proxy usages (LOGPROXY_LOG_D…

### PR DESCRIPTION
**Pull request by thebaptiste**
…IRECTORY env variable is used by default)

(mfserv part of https://github.com/metwork-framework/resources/issues/54)

### CODE REVIEW

File: .metwork-framework/cheatsheet.md [hunk 1]
```diff
@@ -92,7 +92,7 @@ As `mfserv` unix user:
 | --- | --- |
 | **`config.ini`** | main plugin configuration file |
 | **`Makefile`** | build configuration file (you probably don't need to touch this unless you have specific build directives to add to the `custom::` target) |
-| `crontab` | plugin crontab (use something like `* * * * * {{MFSERV_HOME}}/bin/cronwrap.sh -l -e -- plugin_wrapper {YOUR_PLUGIN_NAME} {YOUR_COMMAND} >>{{MODULE_RUNTIME_HOME}}/log/your_command.log 2>&1` for each line to execute your command in your plugin environement) |
+| `crontab` | plugin crontab (use something like `* * * * * {{MFSERV_HOME}}/bin/cronwrap.sh -l -e -- plugin_wrapper {YOUR_PLUGIN_NAME} {YOUR_COMMAND} >>your_command.log 2>&1` for each line to execute your command in your plugin environement) |
```
Comment (thefab) [inline .metwork-framework/cheatsheet.md]: this doc has to be fixed as it does not use --log-capture-to

File: doc/mfserv_miscellaneous.md [hunk 2]
```diff
@@ -154,7 +154,7 @@ If you need to execute your `cron` command in the Metwork context, you should us
 {% raw %}
 {{MFSERV_HOME}}/bin/cronwrap.sh --lock --low "find {{MFMODULE_RUNTIME_HOME}}/var/archive/ -type f -mtime +5 -exec rm -f {} \;" >/dev/null 2>&1
 
-{{MFSERV_HOME}}/bin/cronwrap.sh -- plugin_wrapper [your_plugin_name]  [your_sh_command] >{{MFMODULE_RUNTIME_HOME}}/log/[your_log_filename] 2>&1
+{{MFSERV_HOME}}/bin/cronwrap.sh -- plugin_wrapper [your_plugin_name]  [your_sh_command] >[your_log_filename] 2>&1
```
Comment (thefab) [inline doc/mfserv_miscellaneous.md]: this doc has to be fixed as it does not use --log-capture-to

File: .metwork-framework/cheatsheet.md (hunk 1 above)
Comment (thebaptiste) [reply .metwork-framework/cheatsheet.md]: Ok. But this file is under resources. We won't fix it in mfserv.

File: doc/mfserv_miscellaneous.md (hunk 2 above)
Comment (thebaptiste) [reply doc/mfserv_miscellaneous.md]: Done

**Pull request merged**